vultr-github/terraform/vultr/modules/workload-cluster/main.tf

```
@@ -0,0 +1,202 @@
+resource "vultr_kubernetes" "cluster" {
+  region  = var.cluster_region
+  label   = var.cluster_name
+  version = "v1.30.6+1"
+
+  node_pools {
+    plan          = var.node_type
+    label         = "${var.cluster_name}-node-pool"
+    auto_scaler   = true
+    node_quantity = var.node_count
+    min_nodes     = var.node_count # tonumber() is used for a string token value
+    max_nodes     = var.node_count # tonumber() is used for a string token value
```

```suggestion
    min_nodes     = var.node_count
    max_nodes     = var.node_count
```

Doesn't look like `tonumber` is used